ci/ansible/roles/subscription-manager/tasks/main.yaml

```diff
@@ -22,6 +22,7 @@
     username: "{{ rhn_username }}"
     password: "{{ rhn_password }}"
     pool: "{{ rhn_pool }}"  # e.g. rhn_pool='^SKU Name$'
+  ignore_errors: yes
```

Comment:
I think this is safe to do for all rhel jobs because if the subscription _really_ did fail, then enabling the repos would fail and the job would abort.